src/chrome/content/rules/HurricaneElectric.xml

```diff
@@ -1,43 +1,27 @@
 <!--
 	Other Hurricane Electric rulesets:
-
 		- TunnelBroker.xml
 
-
-
-	Nonfunctional subdomains:
-
-		- bgp.he.net	(refused)
-		- man.he.net	(redirects to www.he.net, valid cert)
-
-
-	bgp.he.net and faq.he.net don't support SSL at the moment
-
-
-	Problematic domains:
-
-		- (www.)hu.com *
-		- hurricaneelectric.com *
-
-	* Alternative domains, same content, using he.net certificate
-
+	www.hurricaneelectric.net does not exist.
```

Comment:
Please keep a comment for the mismatch on `hurricaneelectric.net`.

Reply:
:+1: